src/main/java/com/titarenko/service/ReaderImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.titarenko.service;
+
+import com.titarenko.model.Employee;
+import com.titarenko.model.Gender;
+
+import java.util.*;
+
+public class ReaderImpl implements Reader {
+
+    Scanner scanner = new Scanner(System.in);
+    ConsoleWriterImpl consoleWriter = new ConsoleWriterImpl();
+
+    @Override
+    public Employee readEmployee() {
+        Employee employee = new Employee();
+
+        consoleWriter.writeToOutputStream("name: ");
+        employee.setName(scanner.nextLine());
+
+        consoleWriter.writeToOutputStream("sex (M, F or OTHER): ");
+        employee.setSex(Gender.valueOf(scanner.nextLine()));
+
+        consoleWriter.writeToOutputStream("position: ");
+        employee.setPosition(scanner.nextLine());
+
+        consoleWriter.writeToOutputStream("salary (in $): ");
+        employee.setSalary(Double.parseDouble(scanner.nextLine()));
+
+        consoleWriter.writeToOutputStream("date (dd.mm.yyyy): ");
+        String[] date = scanner.nextLine().split("\\.");
+        int d = Integer.parseInt(date[0]);
```

Comment:
Do you think it might be better to create two separate methods:
```
Gender readGender();
LocalDateTime readDate();
```
and validate inputs there?